_config.yml

```diff
@@ -1,4 +1,4 @@
 name: Your New Jekyll Site
-markdown: redcarpet
-pygments: true
+markdown: kramdown
+highlighter: pygments
```

Comment:
Add a break line between this and `baseurl` to allow this block to be easier to modify in the future.